bears/jinja2/Jinja2Bear.py

```diff
@@ -112,11 +112,16 @@ class Jinja2Bear(LocalBear):
 
     VARIABLE_REGEX = re.compile(
         r'(?P<open>{{)(?P<content>.*?)(?P<close>}})')
+    STATEMENT_REGEX = re.compile(
+        r'(?P<open>{%[+-]?)'
+        # Ignore PycodestyleBear (E501) LineLengthBear
+        r'(?P<content>(?![+-]?\s*for)(?![+-]?\s*if)(?![+-]?\s*endfor)(?![+-]?\s*endif).*?)'
+        r'(?P<close>[+-]?%})')
```

Comment:
Isn't this covered by `CONTROL_START_REGEX` and `CONTROL_END_REGEX`?

Reply:
No this was not covered b `CONTROL_START_REGEX` and `CONTROL_END_REGEX`. In fact if I do not include this, all my tests for statement blocks would fail.

Reply:
Can you reuse the `CONTROL_..._REGEX` variables in your `STATEMENT_REGEX`?